src/features/subReddits/subreddit.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { useDispatch } from "react-redux";
+import { selectSubreddit } from "../posts/postsSlice";
+import { useNavigate } from "react-router-dom";
+import {
+  ListItemButton,
+  Avatar,
+  ListItemAvatar,
+  Typography,
+  ListItemText,
+} from "@mui/material";
+
+
+/* - takes the following props:
+    - title
+    - image
+- onclick renders a new postlist
+*/
+
+export const Subreddit = ({ subreddit }) => {
+  const thumbnail = subreddit.data.icon_img;
+  const subscribers = subreddit?.data?.subscribers?.toLocaleString("en-US");
+  const dispatch = useDispatch();
+  const navigate = useNavigate();
+  const onClickHandler = () => {
+    dispatch(selectSubreddit(subreddit.data.url.slice(3)));
+    navigate(`${subreddit.data.url}`);
+  };
+
+  return (
+    <li>
+      {subreddit && (<ListItemButton
+        divider
+        aria-label="select subreddit"
+        onClick={onClickHandler}
+        disableRipple
+      >
+        <ListItemAvatar minwidth={"150rem"}>
+          <Avatar
+            src={
+              thumbnail ||
+              "https://external-preview.redd.it/iDdntscPf-nfWKqzHRGFmhVxZm4hZgaKe5oyFws-yzA.png?width=640&crop=smart&auto=webp&s=bfd318557bf2a5b3602367c9c4d9cd84d917ccd5"
```

Comment:
Should be a constant somewhere, beware of hardcoding values like this

Reply:
fixed